integration_tests/test_message_queue/test_message_queue_service.py

```diff
@@ -144,3 +146,30 @@ async def test_incorrect_vhost(mocker, caplog):
     await service.initialize()
 
     assert any("Incorrect vhost?" in rec.message for rec in caplog.records)
+
+
+async def test_initialize_declare_exchange_race_condition():
+    service = MessageQueueService()
+
+    async def keep_declaring_exchanges():
+        i = 0
+        while service._channel is None:
+            await service.declare_exchange(f"exchange_{i}")
+            await asyncio.sleep(0)
+            i += 1
+
+    await asyncio.gather(keep_declaring_exchanges(), service.initialize())
+
+
+async def test_declaring_exchange_without_initialization():
```

Comment:
What was the deal with these tests again? Are we not able to get them to run in travis?

Reply:
We can, but they rely on running the faf-rabbitmq container, which I believe we decided not to want to depend on.
It would be quick to add them back to travis, for example the league service is doing that:
https://github.com/cleborys/faf-league-service/blob/9bec96d5f1b0f9bdf69baff014489fc7b3482a9f/.travis.yml#L27
https://github.com/cleborys/faf-league-service/blob/develop/ci/init-rabbitmq.sh

Reply:
Hmm, yea it might be nice to do that in a separate PR. I can't remember why we decided not to before. I remember we put that stuff in the integration tests because we didn't want people to need to install rabbitmq just to develop the server, but I would think on Travis it would be fine.

Reply:
Ah, I see. I didn't see that distinction between contributors running it and travis running it, that's why I moved it to the integration tests. Moving it to travis would also mean the docker repository won't evict `faf-rabbitmq` because of six months of inactivity :stuck_out_tongue_closed_eyes: 

By the way, "installing rabbitmq" is literally only running the script linked above which pulls the docker image from the repository and adds the appropriate user, so the only drawback I'd see for contributors is that the image takes up some space.

Reply:
Hmm, yea that's not a big deal.